packages/shared/src/components/Attachments/Attachments.tsx

```diff
@@ -156,12 +163,14 @@ export const Attachments = ({
                                             collisionDetection={closestCenter}
                                             onDragStart={handleDragStart}
                                             onDragEnd={handleDragEnd}
+                                            modifiers={[restrictToWindowEdges]}
```

Comment:
Why not restrict to Axis? There's no need for this list to be movable via x-axis

Reply:
I fiddled around with this and if I fix it to y axis it results in a worse UX, at least that was my experience. I like that I can slightly move the drag overlay outside its axis, feels a bit more natural.

What do you think?